todo.js

```diff
@@ -1,44 +1,29 @@
-"use strict";
-const todoItems = [];
-
-const id = 0;
+let id = 0;
 
 function getId() {
-  id++;
-  return id;
+    id++;
+    return id;
 }
-function addTodo(newTodo) {
-  const todo = {
-    text,
-    checked: false,
-    id: Date.now(),
-  };
-  todoItems.push(todo);
```

Comment:
The task was to avoid any mutation of data.
https://lorenstewart.me/2017/01/22/javascript-array-methods-mutating-vs-non-mutating/